pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeConvert.java

```diff
@@ -83,4 +99,67 @@ public Object dateTimeConvert(String timeValueStr, String inputFormatStr, String
       return _outputFormatSpec.fromMillisToFormat(roundedTimeValueMs);
     }
   }
+
+  @ScalarFunction
+  public Object dateTimeConvert(String timeValueStr, String inputFormatStr, String outputFormatStr,
```

Comment:
It is almost the same as the other method. Can we introduce a helper method with nullable `bucketingTimeZone`, and make both the scalar function call the helper method

Reply:
I kept two methods but extracted commons bits into helper functions.